app/client/jsx/Room.jsx

```diff
@@ -1,275 +1,314 @@
-import React, { Component } from 'react'
-import { connect } from 'react-redux'
-import reducers from './reducers.jsx'
-import { Redirect } from 'react-router-dom'
-import { Beforeunload } from 'react-beforeunload'
-import ReactMarkdown from 'react-markdown'
-import Modal from 'react-modal';
-import ModalFactory from './ModalFactory.jsx'
-import JitsiVideo from './JitsiVideo.jsx'
-import ArtRoom from './ArtRoom.jsx'
-import IFrameRoom from './IFrameRoom.jsx'
-import ImageMapRoom from './ImageMapRoom.jsx'
-import Door from './Door.jsx'
-import Adventure from './Adventure.jsx'
-import Navigation from './Navigation.jsx'
-import { PokeNotification } from './Poke.jsx'
-import { WebSocketApi } from './WebAPI.jsx'
-import LocalStorage from './LocalStorage.jsx'
-import Config from './Config.jsx'
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import reducers from "./reducers.jsx";
+import { Redirect } from "react-router-dom";
+import { Beforeunload } from "react-beforeunload";
+import ReactMarkdown from "react-markdown";
+import Modal from "react-modal";
+import ModalFactory from "./ModalFactory.jsx";
+import JitsiVideo from "./JitsiVideo.jsx";
+import ArtRoom from "./ArtRoom.jsx";
+import IFrameRoom from "./IFrameRoom.jsx";
+import ImageMapRoom from "./ImageMapRoom.jsx";
+import Door from "./Door.jsx";
+import Adventure from "./Adventure.jsx";
+import Navigation from "./Navigation.jsx";
+import { PokeNotification } from "./Poke.jsx";
+import { WebSocketApi } from "./WebAPI.jsx";
+import LocalStorage from "./LocalStorage.jsx";
+import Config from "./Config.jsx";
+import { ChatStreamRoom } from "./ChatStreamRoom.jsx";
 
```

Comment:
The diff in this file is a little confusing because of what I presume is linting…is there any chance you could commit without the linting? That way our history will be cleaner and reviews will be easier!